packages/snaps-utils/src/node.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/export */
+
+export * from '.';
```

Comment:
This is a nice pattern, we should let everyone in the team know about how exports will work now.

Reply:
That was my plan after merging this yeah.